### PR TITLE
SEO-173239 WPF Chart Legend

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, and changed image alt.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/Charts/Legend.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ chart.Legend = new ChartLegend();
 
 {% endtabs %}
 
-![Legend in WPF Chart](Legend_images/wpf-chart-legend.png)
+![Legend in WPF Chart](legend_images/wpf-chart-legend.png)
```
Comment (Girija-Rajendran) [inline wpf/Charts/Legend.md:38]: Changed this name in folder

Comment (Yvone-Atieno) [reply wpf/Charts/Legend.md:38]: the folder is changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**